content/blog/new-powershell-library-for-the-hpe-greenlake-cloud-platform.md

```diff
@@ -0,0 +1,358 @@
+* To assign and unassign devices to applications:
+
+  * **Set-HPEGLDeviceApplication**: to attach devices to an application instance.
+
+  > **Note**: Assigning devices to an application instance is the process of attaching devices to an application in a region, so that these devices become visible and managed in the application instance by users.
+
+* To create and delete API application credentials:
+
+  * **New-HPEGLAPIcredential**: to create an API credential for an application instance. 
+
+  > **Note**: With the HPE GreenLake platform, developers can make API calls on any application instance as long as they have the API credentials which consist of a client ID, client secret and connectivity endpoint. 
+  >
+  > **Note**: When API credentials are created, they are automatically stored in the global variable **$HPEGreenLakeSession.apiCredentials**. This global variable is accessible as long as the PowerShell console is active and **Disconnect-HPEGL** has not been used. In other words, as long as your session is active. This data is sensitive because it contains all you need to make API calls with an application API such as Compute Ops Management, Data Services Cloud Console, etc. To get a more complete example of how to deeply interact with Compute Ops Management, see [Interaction-with-COM_Sample](https://github.com/HewlettPackard/POSH-HPEGreenLake/blob/master/Samples/Interaction-with-COM_Sample.ps1) script sample.
+  >
+  > **Note**: To store the API credentials beyond the duration of the session, the cmdlet provides a **\-Location** parameter. This parameter can be used with **New-HPEGLAPIcredential** to save the API credentials in a directory and the **\-Encrypt** parameter can be utilized to encrypt the API credentials before exporting the JSON file into the designated *Location* directory.
+
+  * **Remove-HPEGLAPIcredential**: to delete an API credential of an application instance.
+
+  > **Note**: Once API credentials are deleted, access to the application instance API is lost permanently. This is because API credentials are used to authenticate and authorize access to the API. When the credentials are deleted, the corresponding API client ID is also invalidated, which means that any requests made using those credentials will be rejected by the API server.
+
+## Managing subscriptions with the HPE GreenLake PowerShell Library
+
+The main cmdlets for managing subscriptions in the HPE GreenLake Platform are the followings:
+
+* To get information about subscriptions: 
+
+  * **Get-HPEGLDeviceSubscription**: to get information about your device subscriptions available in your HPE GreenLake account. Several parameters can be used to display the subscriptions with available quantity, the subscriptions that are expired or not expired, etc. You can combine parameters to obtain for example only the subscription keys that have not expired and for which there are still licenses available: **Get-HPEGLDeviceSubscription -Notexpired -Available**
```

Comment:
Recommend rephrasing the last sentence. The phrase "for example" is out of place. Try this instead:
"For example, you can combine parameters to obtain only the subscription keys that have not expired and for which there are still licenses available:"